vayesta/solver/coupling.py

```diff
@@ -280,52 +283,181 @@ def _integrals_for_extcorr(fragment, fock):
         govovbb = eris[ob,vb,ob,vb]
         fov = (fova, fovb)
         govov = (govovaa, govovab, govovbb)
-    return fov, govov
+        gvvov = None
+        gooov = None
+        govoo = None
+        raise NotImplementedError
+    return fov, govov, gvvov, gooov, govoo
 
+def _get_delta_t_for_extcorr(fragment, fock, solver, include_t3v=True):
+    ''' Make T3 and T4 correction to CCSD wave function for given fragment.
```

Comment:
Docstrings with `"""` and no space before first letter please

Reply:
Done